scripts/viz/param_sweep_latent_space_viz.R

```diff
@@ -0,0 +1,321 @@
+# Pan-Cancer Variational Autoencoder
+# Gregory Way 2018
+#
+# Visualizing results of tybalt and adage parameter sweeps over z dimensions
+#
+# First, to summarize the parameter sweep into a single file run:
+#
+#     python scripts/summarize_paramsweep.py
+#               --results_directory "z_param_sweep_tybalt"
+#               --output_filename "z_parameter_sweep_tybalt_full_results.tsv"
+#
+# Also perform the same for the ADAGE sweep
+#
+# Output:
+# Several figures describing the results of a parameter sweep over latent space
+# dimensionality for both Tybalt and ADAGE models
+
+library(dplyr)
```

Comment:
Since you use `dplyr::` quite a bit here, I'm assuming that this is just for the magrittr pipe? If so, can do:

```R
`%>%` <- dplyr::`%>%`
```